nvtabular/ops/categorify.py

```diff
@@ -421,6 +422,14 @@ def get_embedding_sizes(self, columns):
             self.categories, columns, self.num_buckets, self.freq_threshold, self.max_size
         )
 
+    def inference_initialize(self, columns: ColumnNames, model_config: dict) -> Optional[Operator]:
+        # on the first transform call we load up categories from disk, which can
+        # take multiple seconds. preload this data by running an empty dataframe through
```

Comment:
This feels like a long way around to load the data during initialization. Is there something about loading the categories from disk that we wouldn't want to do during the normal `__init__` method for non-inference use cases?

Reply:
agreed - this method will be swapped out in a future PR